.circleci/config.yml

```diff
@@ -1,14 +1,194 @@
+      - run:
+          name: Build graphql server docker image
+          command: docker build . -f build/Dockerfile -t ${REPONAME}/${GRAPHQL_SERVER_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/backend/graphql-server
+      - run:
+          name: Save graphql server docker image
+          command: docker save -o workspace/graphql-server.tar ${REPONAME}/${GRAPHQL_SERVER_IMAGE}:${IMGTAG}
+      - run:
+          name: Build auth server docker image
+          command: docker images && docker build . -f Dockerfile -t ${REPONAME}/${AUTHENTICATION_SERVER_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/backend/auth
+      - run:
+          name: Save auth server docker image
+          command: docker save -o workspace/auth-server.tar ${REPONAME}/${AUTHENTICATION_SERVER_IMAGE}:${IMGTAG}
+      - run:
+          name: Build subscriber docker image
+          command: docker build . -f build/Dockerfile -t ${REPONAME}/${SUBSCRIBER_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/backend/subscriber
+      - run:
+          name: Save subscriber docker image
+          command: docker save -o workspace/subscriber.tar ${REPONAME}/${SUBSCRIBER_IMAGE}:${IMGTAG}
+      - run:
+          name: Build self-deployer docker image
+          command: docker build . -f build/Dockerfile -t ${REPONAME}/${SELF_DEPLOYER_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/backend/tools/self-deployer
+      - run:
+          name: Save self-deployer docker image
+          command: docker save -o workspace/self-deployer.tar ${REPONAME}/${SELF_DEPLOYER_IMAGE}:${IMGTAG}
+      - run:
+          name: Build frontend docker image
+          command: docker build . -f Dockerfile -t ${REPONAME}/${FRONTEND_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/frontend
+      - run:
+          name: Save frontend docker image
+          command: docker save -o workspace/frontend.tar ${REPONAME}/${FRONTEND_IMAGE}:${IMGTAG}
+      - persist_to_workspace:
+          root: workspace
+          paths:
+            - ${GRAPHQL_SERVER_IMAGE}.tar
+            - ${SUBSCRIBER_IMAGE}.tar
+            - ${SELF_DEPLOYER_IMAGE}.tar
+            - ${AUTHENTICATION_SERVER_IMAGE}.tar
+            - ${FRONTEND_IMAGE}.tar
+
+  docker-push:
     machine:
-      image: ubuntu-1604:201903-01
-    working_directory: ~/go/src/github.com/litmuschaos/litmus
+      image: circleci/classic:201808-01
     environment:
-      GO111MODULE: "on"
+      IMGTAG: ci
     steps:
-      - run : |
-          cd && source /opt/circleci/.nvm/nvm.sh && nvm install v10 && nvm alias default 10 && wget https://dl.google.com/go/go1.14.6.linux-amd64.tar.gz && sudo tar -xvf go1.14.6.linux-amd64.tar.gz && sudo rm -rf /usr/local/go && sudo mv go /usr/local && export GOROOT=/usr/local/go
+      - attach_workspace:
+          at: /tmp/workspace
+      - run: |
+          cat /tmp/workspace/env-vars >> $BASH_ENV
```

Comment:
I think here source of bash-env is required

Reply:
https://github.com/litmuschaos/litmus/pull/1980/files#r485330275